src/Features/Core/Portable/Completion/CompletionRules.cs

```diff
@@ -65,9 +83,32 @@ public virtual bool MatchesFilterText(CompletionItem item, string filterText, Co
                 return false;
             }
 
-            var patternMatcher = this.GetPatternMatcher(_completionService.GetCultureSpecificQuirks(filterText));
+            return GetMatch(item, filterText) != null;
+        }
+
+        protected PatternMatch? GetMatch(CompletionItem item, string filterText)
+        {
+            var patternMatcher = this.GetPatternMatcher(_completionService.GetCultureSpecificQuirks(filterText), CultureInfo.CurrentCulture);
             var match = patternMatcher.GetFirstMatch(_completionService.GetCultureSpecificQuirks(item.FilterText));
-            return match != null;
+
+            if (match != null)
+            {
+                return match;
+            }
+
+            // Fallback for Turkish:
```

Comment:
Comment now outdated.  Something like `If the CurrentCulture didn't match, fall back to en-US, since many programmers use English identifiers`